packages/compass-connections/src/stores/connections-store.tsx

```diff
@@ -163,17 +163,17 @@ export function useConnections({
   onConnectionAttemptStarted,
   __TEST_INITIAL_CONNECTION_INFO,
 }: {
-  onConnected: (
+  onConnected?: (
     connectionInfo: ConnectionInfo,
     dataService: DataService
   ) => void;
-  onConnectionFailed: (
+  onConnectionFailed?: (
```

Comment:
I'm not sure why these methods should be optional, is this for testing purposes?

Reply:
As soon as I will move the telemetry from compass-home to compass-connections, they will not be used anymore by compass-home. Eventually we will drop them from compass-web too and remove completely, this type just anticipates that